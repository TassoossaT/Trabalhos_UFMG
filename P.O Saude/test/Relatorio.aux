\relax 
\providecommand\babel@aux[2]{}
\@nameuse{bbl@beforestart}
\catcode `"\active 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\babel@aux{brazil}{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introdução}{2}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}Metodologia}{2}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Formulação do Problema}{2}{subsection.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Dados Utilizados}{4}{subsection.2.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}Conclusão}{4}{section.3}\protected@file@percent }
\newlabel{LastPage}{{3}{4}{Conclusão}{page.4}{}}
\gdef\lastpage@lastpage{4}
\gdef\lastpage@lastpageHy{4}
\gdef \@abspage@last{4}
