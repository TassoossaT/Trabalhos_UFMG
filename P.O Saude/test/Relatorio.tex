\documentclass[a4paper,12pt]{article}
\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage{geometry}

\usepackage[brazil]{babel}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{tocloft}
\usepackage{fancyhdr}
\usepackage{lastpage}
\usepackage{titling}

\geometry{a4paper, margin=1in}

\pretitle{
    \begin{center}
    \includegraphics[width=0.4\textwidth]{principal_completa_ufmg.jpg}\\[1em]
    \textbf{\fontsize{12}{14}\selectfont UNIVERSIDADE FEDERAL DE MINAS GERAIS}\\
    \large Faculdade de Engenharia de Produção\\[4em]
}
\posttitle{\end{center}\vfill\large\date{\today}}

\title{
    \begin{center}
    \textbf{\fontsize{12}{14} \selectfont Relatório de Apresentação e Modelo de Pesquisa Operacional na Saúde}
    \end{center}
}
\author{Tasso A. T. Pimmenta \\ 2021072198}

\date{\today}
\setcounter{tocdepth}{2}



\begin{document}
\maketitle


\begin{abstract}
Este relatório apresenta um modelo de pesquisa operacional aplicado à saúde, focando na localização e facilidades. 
O objetivo é otimizar a distribuição de recursos e melhorar o acesso aos serviços de saúde.
Foi apresentado diversos algoritmos da literatura e o que cada um traz como objetivo e como é aplicado.
Meu trabalho se concentra especificamente em um algoritmo hierarquico para representar os hospitais, 
com o objtivo de minimizar os custos.
\end{abstract}

\newpage

\tableofcontents

\section{Introdução}
A aplicação de modelos de pesquisa operacional na área da saúde tem por objetvo otimizar a distribuição de recursos.
Os modelos mais comuns de localização de facilidades são:
\begin{itemize}
    \item \textbf{Set covering}: visa atender toda a demanda possível com o menor número de facilidades.
    \item \textbf{Maximal covering}: visa atender a maior parte da demanda possível com facilidades limitadas.
    \item \textbf{P-median}: escolhe a localização com base na densidade da demanda.
    \item \textbf{P-center}: escolhe a localização para reduzir a distância máxima entre a facilidade e a demanda.
    \item \textbf{Fixed charger}: visa atender a demanda com o menor custo possível.
    \item \textbf{Hierarchical}: visa atender a demanda com o menor custo possível, mas com uma hierarquia de facilidades.
\end{itemize}

\section{Metodologia}
Descrevemos a metodologia utilizada para desenvolver o modelo de localização de facilidades. 
Inclui a formulação do problema, as variáveis envolvidas e as restrições consideradas.

A escolha do modelo foi um baseado no modelo Hierárquico, onde cada nivel representa uma categoria de unidade de saúde.
O modelo foi contruido de forma a ser tanto um modelo de alocação quanto como de distribuição, 
onde se minimiza os custos de atendimento e funcionamento, de modo onde se for mais barato é permitido contruir unidades novas.

\subsection{Formulação do Problema}
O modelo hierárquico de localização de facilidades é formulado como um problema de programação linear inteira mista. A seguir, apresentamos a formulação matemática do problema:

\textbf{Variáveis de Decisão:}
\begin{itemize}
    \item $y_{i,j}$: variável binária que indica se a demanda $i$ é atendida pela unidade de saúde $j$.
    \item $y1_{j}$, $y2_{j}$, $y3_{j}$: variáveis binárias que indicam se a unidade de saúde $j$ nos níveis 1, 2 e 3 está aberta, respectivamente.
    \item $u1_{p,i,j}$, $u2_{j,j2}$, $u3_{j2,j3}$: variáveis contínuas que representam o fluxo de pacientes entre os níveis de unidades de saúde.
    \item $l1_{e,l}$, $l2_{e,l}$, $l3_{e,l}$: variáveis contínuas que representam a alocação de equipes de saúde nas unidades de saúde.
\end{itemize}

\textbf{Função Objetivo:}
Minimizar o custo total, que inclui custos de travessia, custos fixos e variáveis das unidades de saúde, e custos de equipe:
\[
\begin{aligned}
\text{Minimizar} \quad & \sum_{p \in P} \sum_{i \in I} \sum_{j \in L[1]} D1_{i,j} \cdot TC1_{i,j} \cdot u1_{p,i,j} + \\
& \sum_{j \in L[1]} \sum_{j2 \in L[2]} D2_{j,j2} \cdot TC2_{j,j2} \cdot u2_{j,j2} + \\
& \sum_{j2 \in L[2]} \sum_{j3 \in L[3]} D3_{j2,j3} \cdot TC3_{j2,j3} \cdot u3_{j2,j3} + \\
& \sum_{j \in EL[1]} FC1_{j} \cdot y1_{j} + \sum_{j2 \in EL[2]} FC2_{j2} \cdot y2_{j2} + \sum_{j3 \in EL[3]} FC3_{j3} \cdot y3_{j3} + \\
& \sum_{j \in CL[1]} \sum_{e \in E[1]} CE1_{e} \cdot y1_{j} + \sum_{j2 \in CL[2]} \sum_{e \in E[2]} CE2_{e} \cdot y2_{j2} + \sum_{j3 \in CL[3]} \sum_{e \in E[3]} CE3_{e} \cdot y3_{j3} + \\
& \sum_{p \in P} \sum_{i \in I} \sum_{j1 \in L[1]} D1_{i,j1} \cdot TC1_{i,j1} \cdot u1_{p,i,j1} \cdot VC1_{p,j1} + \\
& \sum_{p \in P} \sum_{j \in L[1]} \sum_{j2 \in L[2]} D2_{j,j2} \cdot TC2_{j,j2} \cdot u2_{j,j2} \cdot VC2_{p,j2} + \\
& \sum_{p \in P} \sum_{j2 \in L[2]} \sum_{j3 \in L[3]} D3_{j2,j3} \cdot TC3_{j2,j3} \cdot u3_{j2,j3} \cdot VC3_{p,j3}
\end{aligned}
\]

\textbf{Restrições:}
\begin{itemize}
    \item Cobertura universal:
    \[
    \sum_{p \in P} u1_{p,i,j} = \sum_{p \in P} W_{i,p} \cdot y_{i,j} \quad \forall i \in I, \forall j \in L[1]
    \]
    \item Cada demanda é atendida por uma unidade de saúde:
    \[
    \sum_{j \in L[1]} y_{i,j} = 1 \quad \forall i \in I
    \]
    \item Fluxo entre níveis de unidades de saúde:
    \[
    \sum_{j2 \in L[2]} u2_{j,j2} = O1_{j} \cdot \sum_{p \in P} \sum_{i \in I} u1_{p,i,j} \quad \forall j \in L[1]
    \]
    \[
    \sum_{j3 \in L[3]} u3_{j2,j3} = O2_{j2} \cdot \sum_{j \in L[1]} u2_{j,j2} \quad \forall j2 \in L[2]
    \]
    \item Capacidade das unidades de saúde existentes:
    \[
    \sum_{i \in I} u1_{p,i,j} \leq C1_{p,j} \quad \forall j \in EL[1], \forall p \in P
    \]
    \[
    \sum_{j \in L[1]} u2_{j,j2} \leq C2_{j2} \quad \forall j2 \in EL[2]
    \]
    \[
    \sum_{j2 \in L[2]} u3_{j2,j3} \leq C3_{j3} \quad \forall j3 \in EL[3]
    \]
    \item Capacidade das novas unidades de saúde:
    \[
    \sum_{i \in I} u1_{p,i,j} \leq C1_{p,j} \cdot y1_{j} \quad \forall j \in CL[1], \forall p \in P
    \]
    \[
    \sum_{j \in L[1]} u2_{j,j2} \leq C2_{j2} \cdot y2_{j2} \quad \forall j2 \in CL[2]
    \]
    \[
    \sum_{j2 \in L[2]} u3_{j2,j3} \leq C3_{j3} \cdot y3_{j3} \quad \forall j3 \in CL[3]
    \]
    \item Limite no número de novas unidades de saúde:
    \[
    \sum_{j \in CL[1]} y1_{j} \leq U[1]
    \]
    \[
    \sum_{j2 \in CL[2]} y2_{j2} \leq U[2]
    \]
    \[
    \sum_{j3 \in CL[3]} y3_{j3} \leq U[3]
    \]
\end{itemize}

\subsection{Dados Utilizados}
os dados que uso no meu trablho são os dados reais da cidade de Belo Horizonte.
então a demanda é a quantidade de pessoas, os pontos de demandas os bairros ou as zonas sensitarias.
A parte que não é exatamente que são dados reais são as localidades candidatas a contrução de novas unidades de saúde.
Sendo então uma escolha arbitraria de onde posso permitir a contrução de novas unidades de saúde.

\section{Conclusão}

O modelo hierárquico de localização de facilidades é uma abordagem eficaz para otimizar a distribuição de recursos na área da saúde.
Ainda é necesseário um refino no modelo, ainda mais levando em conta as dficuldades de usar e avaliar os dados reais.
Existe uma imprecisão muito grande nos dados disponiveis, existe erro nos disponibilizados e falta muitos. 
por isso ainda não consigo avaliar o modelo com exatidão e ver o que ele me da como resposta otima.
\end{document}
