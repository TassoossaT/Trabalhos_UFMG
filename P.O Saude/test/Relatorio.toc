\babel@toc {brazil}{}\relax 
\contentsline {section}{\numberline {1}Introdução}{2}{section.1}%
\contentsline {section}{\numberline {2}Metodologia}{2}{section.2}%
\contentsline {subsection}{\numberline {2.1}Formulação do Problema}{2}{subsection.2.1}%
\contentsline {subsection}{\numberline {2.2}Dados Utilizados}{4}{subsection.2.2}%
\contentsline {section}{\numberline {3}Conclusão}{4}{section.3}%
