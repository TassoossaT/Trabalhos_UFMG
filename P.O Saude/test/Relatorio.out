\BOOKMARK [1][-]{section.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000\347\000\343\000o}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000M\000e\000t\000o\000d\000o\000l\000o\000g\000i\000a}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{\376\377\000F\000o\000r\000m\000u\000l\000a\000\347\000\343\000o\000\040\000d\000o\000\040\000P\000r\000o\000b\000l\000e\000m\000a}{section.2}% 3
\BOOKMARK [2][-]{subsection.2.2}{\376\377\000D\000a\000d\000o\000s\000\040\000U\000t\000i\000l\000i\000z\000a\000d\000o\000s}{section.2}% 4
\BOOKMARK [1][-]{section.3}{\376\377\000C\000o\000n\000c\000l\000u\000s\000\343\000o}{}% 5
